lib/device-log/ios-log.js

```diff
@@ -199,16 +203,30 @@ class IOSLog {
   }
 
   async stopCapture () {
-    logger.debug('Stopping iOS log capture');
-    if (this.proc && this.proc.isRunning) {
+    if (this.proc) {
+      if (this.udid) {
```

Comment:
can it be that udid is not set?

Reply:
Checking `this.udid` checks if it's a real device (which `idevicesyslog` runs on)